pkg/util/watchdog/watchdog_test.go

```diff
@@ -216,12 +216,16 @@ var _ = Describe("SimpleWatchdog", func() {
 			close(doneCh)
 		}()
 		Expect(watchdog.HasTicked(false)).Should(BeFalse())
-		Consistently(hasTicked).ShouldNot(Receive())
+		Consistently(func(g Gomega) {
+			g.Expect(hasTicked).ShouldNot(Receive())
```

Comment:
isn't it the same as
```go
Consistently(hasTicked, "100ms", "10ms").ShouldNot(Receive())
```

Reply:
Yes, changed it since it should works well for channel but in case of function( eventually few lines below it won't work)